lib/system/sysio.nim

```diff
@@ -52,6 +52,8 @@ proc c_ferror(f: File): cint {.
   importc: "ferror", header: "<stdio.h>", tags: [].}
 proc c_setvbuf(f: File, buf: pointer, mode: cint, size: csize): cint {.
   importc: "setvbuf", header: "<stdio.h>", tags: [].}
+proc c_ftruncate(f: FileHandle, len: clong): cint {.
```

Comment:
ftruncate takes an `off_t` - what `off_t` is depends on platform and compile flags - like this, results may be mixed when setting file sizes >2gb as nim will silently truncate the value.